src/Scene/LayersConfiguration.js

```diff
@@ -18,6 +18,12 @@ function LayersConfiguration() {
     this.geometryLayers = [];
     this.colorLayers = [];
     this.elevationLayers = [];
+    this.lightingLayers = [];
```

Comment:
On a more serious tone: even if layers are losely defined actually, I don't believe their purpose is to store sun position.

My suggestion to store it in `Globe` is simply based on the fact that this class all globe related parameters/var.